### DISCUSSION TITLE
What Feature Should Demolition Engine Have?

### DISCUSSION THREAD

**Discussion by RealDealJinx** [DEMOLITIONDON96/Demolition-Engine#53] [Polls]
funni i stole from @DEMOLITIONDON96 >:)

**Comment by Theoyeah** [comment]
kade modcharts would be perfect

**Comment by ThatOneSillyGuyDon** [comment]
# Results: Time to add support for Kade Modcharts, oh boy

**Reply by Theoyeah** [reply]
# We gonna fucking die **LOL**

**Reply by RealDealJinx** [reply]
did that and it failed so no more kade modcharts
